requests-for-comments/2022-11-09-nail-distribution.md

```diff
@@ -0,0 +1,80 @@
+# Request for comments: Nail Content Distribution
+
+## Problem statement
+
+We don't have a clear view on how to maximize the impact of our content through good distribution. We have tried (and are trying) various initiatives, but it's not particularly coordinated or structured, so it's hard to say what's definitely working well and what's not for the most part. 
+
+It's rational to go deep on a smaller selection of distribution channels and trying to do them really well, rather than blasting out content onto every platform imaginable in a shallow way. We've found this quality > quantity approach works well for us when it comes to writing the content itself. 
+
+Another problem is that it's not clear whose responsibility on the team content distribution is, so it ends up being a bit ad hoc. This is easy to solve though. 
+
```

Comment:
Additional point: What are we qualifying as content for distribution? 

My main point here would be that we don't limit this to 'anything on /blog', but also widen in to include customer stories and tutorials, as well as potentially other large docs updates for feature releases. 

Arguably we could also consider adding Github issues and RFCs now that /roadmap is up...

Reply:
I think that question is up to the owner of that channel - they should consider for distribution literally anything that PostHog publishes on the internet?

Reply:
I agree it's up to the owner and the only criteria is... what works?